### PR TITLE
Fine-tune ASCII art

### PR DESCRIPTION
**Pull request by PingXie**
Does it look right?

```
                     ___   ___
                    (   ) (   )
 ___  ___    .---.   | |   | |   ___     .--.    ___  ___
(   )(   )  / .-, \  | |   | |  (   )   /    \  (   )(   )   Valkey 255.255.255 (af1b0de9/1) 64 bit
 | |  | |  (__) ; |  | |   | |  ' /    |  .-. ;  | |  | |
 | |  | |    .'`  |  | |   | |,' /     |  | | |  | |  | |    Running in standalone mode
 | |  | |   / .'| |  | |   | .  '.     |  |/  |  | '  | |    Port: 6379
 | |  | |  | /  | |  | |   | | `. \    |  ' _.'  '  `-' |    PID: 2078115
 ' '  ; '  ; |  ; |  | |   | |  \ \    |  .'.-.   `.__. |
 \   `' /  ' `-'  |  | |   | |   \ .   '  `-' /   ___ | |
   '_.'    `.__.'_. (___) (___ ) (___)  `.__.'   (   )' |   https://valkey.io
                                                  ; `-' '
                                                   .__.'
```

### CODE REVIEW

**Comment by madolson** [comment]
It looks like Ualkey to me. I think the k also looks a bit crooked, I think this looks better:
```
                     ___   ___
                    (   ) (   )
 ___  ___    .---.   | |   | |   ___     .--.    ___  ___
(   )(   )  / .-, \  | |   | |  (   )   /    \  (   )(   )   Valkey 255.255.255 (af1b0de9/1) 64 bit
 | |  | |  (__) ; |  | |   | |  ' /    |  .-. ;  | |  | |
 | |  | |    .'`  |  | |   | |,' /     |  | | |  | |  | |    Running in standalone mode
 | |  | |   / .'| |  | |   | .  '      |  |/  |  | '  | |    Port: 6379
 | |  | |  | /  | |  | |   | |`. \     |  ' _.'  '  `-' |    PID: 2078115
 ' '  ; '  ; |  ; |  | |   | |  \ \    |  .'.-.   `.__. |
  \ `' /   ' `-'  |  | |   | |   \ .   '  `-' /   ___ | |
   \__/    `.__.'_. (___) (___ ) (___)  `.__.'   (   )' |   https://valkey.io
                                                  ; `-' '
                                                   .__.'
```

**Comment by PingXie** [comment]
hmm, good point. I think this looks like what you asked?

```
                     ___   ___
                    (   ) (   )
 ___  ___    .---.   | |   | |   ___     .--.    ___  ___
(   )(   )  / .-, \  | |   | |  (   )   /    \  (   )(   )   Valkey 255.255.255 (f228310f/1) 64 bit
 | |  | |  (__) ; |  | |   | |  ' /    |  .-. ;  | |  | |
 | |  | |    .'`  |  | |   | |,' /     |  | | |  | |  | |    Running in standalone mode
 | |  | |   / .'| |  | |   | .  '.     |  |/  |  | '  | |    Port: 6379
 | |  | |  | /  | |  | |   | | `. \    |  ' _.'  '  `-' |    PID: 2304191
 ' '  ; '  ; |  ; |  | |   | |  \ \    |  .'.-.   `.__. |
 \  `'  /  ' `-'  |  | |   | |   \ .   '  `-' /   ___ | |
   \__/    `.__.'_. (___) (___ ) (___)  `.__.'   (   )' |   https://valkey.io
                                                  ; `-' '
                                                   .__.'
```

**Comment by PingXie** [comment]
actually no. here is another try (also aligned the url link with other text strings):

```
                     ___   ___
                    (   ) (   )
 ___  ___    .---.   | |   | |   ___     .--.    ___  ___
(   )(   )  / .-, \  | |   | |  (   )   /    \  (   )(   )   Valkey 255.255.255 (d93f4b5e/1) 64 bit
 | |  | |  (__) ; |  | |   | |  ' /    |  .-. ;  | |  | |
 | |  | |    .'`  |  | |   | |,' /     |  | | |  | |  | |    Running in standalone mode
 | |  | |   / .'| |  | |   | .  '.     |  |/  |  | '  | |    Port: 6379
 | |  | |  | /  | |  | |   | | `. \    |  ' _.'  '  `-' |    PID: 2312058
 ' '  ; '  ; |  ; |  | |   | |   \ \   |  .'.-.   `.__. |
 \  `'  /  ' `-'  |  | |   | |    \ .  '  `-' /   ___ | |
   \__/    `.__.'_. (___) (___ )  (___) `.__.'   (   )' |    https://valkey.io
                                                  ; `-' '
                                                   .__.'
```

File: src/asciilogo.h [hunk 1]
```diff
@@ -28,17 +28,17 @@
...
+"(   )(   )  / .-, \\  | |   | |  (   )   /    \\  (   )(   )   Valkey %s (%s/%d) %s bit  \n"
 " | |  | |  (__) ; |  | |   | |  ' /    |  .-. ;  | |  | |                               \n"
 " | |  | |    .'`  |  | |   | |,' /     |  | | |  | |  | |    Running in %s mode         \n"
 " | |  | |   / .'| |  | |   | .  '.     |  |/  |  | '  | |    Port: %d                   \n"
 " | |  | |  | /  | |  | |   | | `. \\    |  ' _.'  '  `-' |    PID: %ld                 \n"
-" ' '  ; '  ; |  ; |  | |   | |  \\ \\    |  .'.-.   `.__. |                            \n"
-" \\  `' /   ' `-'  |  | |   | |   \\ .   '  `-' /   ___ | |                             \n"
-"   '_.'    `.__.'_. (___) (___ ) (___)  `.__.'   (   )' |   https://valkey.io\n"
+" ' '  ; '  ; |  ; |  | |   | |   \\ \\   |  .'.-.   `.__. |                            \n"
+" \\  `'  /  ' `-'  |  | |   | |    \\ .  '  `-' /   ___ | |                             \n"
```
Comment (madolson) [inline src/asciilogo.h]:
```suggestion
"  \\ `' /   ' `-'  |  | |   | |    \\ .  '  `-' /   ___ | |                             \n"
```
The V still seems a bit bulged, I think this makes it straight?

Comment (PingXie) [reply src/asciilogo.h]: yeah. I think this is perfect now :-)

**Review by madolson — APPROVED** [review]
The k still irks me a bit, (and sort of want to curn the e a bit), but this seems easily readable and makes me happy.

**Comment by PingXie** [comment]
This is the probably the closest I ever got to "art". I have merged it. Hopefully someone else could carry the baton further 😀.

**Pull request merged**